lib/src/model/discovered_device.dart

```diff
@@ -27,13 +27,18 @@ class DiscoveredDevice extends $DiscoveredDevice {
   @CustomEquality(DeepCollectionEquality())
   final Map<Uuid, Uint8List> serviceData;
 
+  /// Returns the manufacturer specific data part of the advertisement packet.
+  /// The first 2 bytes are the Company Identifier Codes.
+  final Uint8List manufacturerData;
```

Comment:
It's not a function, so it doesn't "return" it, it contains it. Maybe, just like this?

```dart
/// Manufacturer specific data. The first 2 bytes are the Company Identifier Codes.
```

Reply:
Done. Thanks